StepDefinitions/Facebook/HM-3VerifyHomeFieldsSD.js

```diff
@@ -0,0 +1,23 @@
+const { Given, When, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const Homepage = require('../../Pages/Facebook/Homepage');
+
+
+const homepage = new Homepage();
+
+
+Then(/^I verify login email field is enabled$/, async function () {
+    const isEmailFieldEnabled = await homepage.isEmailFieldEnabled();
+    expect(isEmailFieldEnabled, 'Login error is NOT displayed').to.be.true;
+
+
+    and(/^I verify login password field is enabled$/, async function () {
```

Comment:
There shouldn't be any And in SD pages. It should be either Given or When or Then.